code_generator_funcadl_uproot/servicex/templates/transform_single_file.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
This is where we can store files that will be copied directly into the tarball. All the rest of the code is emitted from `func_adl_xAOD`. It makes sense that this file is part of ServiceX since it is intimately tied to our code and is of no use outside of ServiceX